zello-sdk-sample-ptt/src/main/java/com/zello/sdk/sample/ptt/PttActivity.java

```diff
@@ -49,15 +51,16 @@ protected void onCreate(Bundle savedInstanceState) {
         speakerButton = (NoToggleButton)findViewById(R.id.audio_speaker);
         earpieceButton = (NoToggleButton)findViewById(R.id.audio_earpiece);
         bluetoothButton = (NoToggleButton)findViewById(R.id.audio_bluetooth);
-        audioModeView = (View)findViewById(R.id.audio_mode);
+        audioModeView = findViewById(R.id.audio_mode);
         messageStateTextView = (TextView)findViewById(R.id.messageStateTextView);
         selectedContactTextView = (TextView)findViewById(R.id.selectedContactTextView);
 
         // Constrain PTT button size
         pttButton.setMaxHeight(getResources().getDimensionPixelSize(R.dimen.talk_button_size));
 
         Zello.getInstance().configure("com.pttsdk", this, this);
-        audio = Zello.getInstance().getAudio();
+		Zello.getInstance().requestVitalPermissions();
```

Comment:
Not an issue for client and maybe just how GitHub is displaying this, but it looks like there are indentation issues in this source where maybe one dev uses an indentation size of 4 and another uses 8